platform-sdk/docs/platformWiki.md

```diff
@@ -0,0 +1,69 @@
+# Swirlds Platform Wiki
+
+This document contains information about the Swirlds Platform. It is currently a work in progress.
+
+The platform code is split into three categories:
+
+- Core: gossip, consensus, data flow, and various algorithms
+- Base: common utilities and libraries; logging, configuration, metrics, etc.
+- Data: merkle data structures for holding the ledger state
+
+## Core
+
+This code is maintained by the "Platform Hashgraph" team.
+
+- Components
+    - Gossip
+        - Sync gossip algorithm
+        - Out of order gossip algorithm
+    - Hashgraph
+    -  State management
+    - State snapshots
+    - Hashing
+    - State Signing
+    - ISS Detection
+    - Reconnect
+    - Transaction Handling
+    - BLS
+- Event Flow
```

Comment:
Please add a place holder for `Event Intake` under `Event Flow`

Reply:
added